File: dataset/output/GEMINI_gemini-1.5-pro-001/Task51_GEMINI_gemini-1.5-pro-001.java
import java.util.Scanner;

public class Encrypt {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string to encrypt: ");
        String message = scanner.nextLine();

        String encryptedMessage = encrypt(message);
        System.out.println("Encrypted message: " + encryptedMessage);
    }

    public static String encrypt(String message) {
        StringBuilder encrypted = new StringBuilder();
        for (char c : message.toCharArray()) {
            encrypted.append((char) (c + 3)); // Simple Caesar cipher with a shift of 3
        }
        return encrypted.toString();
    }
}

File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task51_GEMINI_gemini-1.5-pro-001.cpp
#include <iostream>
#include <string>

std::string encrypt(const std::string& message);

int main() {
    std::string message;
    std::cout << "Enter a string to encrypt: ";
    std::getline(std::cin, message);

    std::string encryptedMessage = encrypt(message);
    std::cout << "Encrypted message: " << encryptedMessage << std::endl;

    return 0;
}

std::string encrypt(const std::string& message) {
    std::string encrypted = "";
    for (char c : message) {
        encrypted += (char)(c + 3); // Simple Caesar cipher with a shift of 3
    }
    return encrypted;
}